ramalama/cli.py

```diff
@@ -624,6 +624,12 @@ def pull_parser(subparsers):
         default=True,
         help="require HTTPS and verify certificates when contacting registries",
     )
+    parser.add_argument(
+        "--verify",
+        default=True,
```

Comment:
Should we add a flag to ramalama.conf to allow users to always pull without verifying?